### PR TITLE
Chore(SCT-1650): Add MASH development documentation

### PR DESCRIPTION
**Pull request by Renny4Real**
[JIRA](https://hackney.atlassian.net/browse/SCT-1650)

For the MASH work we've done we need a place or way to document the work and keep track of changes.

The MASH work lives across multiple repos/projects and is coded using different languages and libraries e.g googleAPIs, typescript, AWS-CLI, C#, REACT.

Since we’re no longer continuing with the MASH work it would be good to have a place where we’ve documented of all the places to look regarding any part of the MASH workflow and keep a log of all the decisions we've made. For example, deleting an unused MongoDB collection for MASH referrals.

Links to Google resources have been complied into a list and saved in a separate Google document.

### CODE REVIEW

File: docs/mash-referrals/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Last updated: 24th January 2022
+
+For children's social care, we had started development on moving the process of managing and working on MASH referrals into the social care case management system. As of writing this, the programme decided not to continue this work.
+
+We are in the process of re-purposing and cleaning up the code and infrastructure setup for the MASH work. We have not finished deciding what gets removed or re-purposed and because of this, we want to keep track of all the resources related to MASH, where they live and what decisions we have made regarding them.
+
+Decisions should be documented in the [Architecture Decision Records](../decisions).
+
+## Table of MASH related resources
```
Comment (LBHTKarki) [inline docs/mash-referrals/index.md:15]: Jira ticket mentioned work in progress branches. Are those documented in relevant repos or are there actually any of them at the moment?

File: docs/decisions/0004-drop-mash-mongodb-collection.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Date: 2022-01-24
+
+## Status
+
+Accepted
+
+## Context
+
+Early in the MASH development, a MongoDB collection was set up for storing the form data from MASH referrals form data. This was only set up in Staging. However this collection has never been unused and is no longer needed.
```
Comment (LBHTKarki) [inline docs/decisions/0004-drop-mash-mongodb-collection.md:14]: unused -> used?

File: docs/mash-referrals/index.md (hunk 1 above)
Comment (LBHTKarki) [reply docs/mash-referrals/index.md:15]: Great idea to keep track of all this 👍

File: docs/decisions/0004-drop-mash-mongodb-collection.md (hunk 2 above)
Comment (Renny4Real) [reply docs/decisions/0004-drop-mash-mongodb-collection.md:14]: Thanks for spotting 😅

Comment (Renny4Real) [reply docs/decisions/0004-drop-mash-mongodb-collection.md]: Done in 027e9dc

**Pull request merged**